### DISCUSSION TITLE
Cisco ASR9K netflow decoding

### DISCUSSION THREAD

**Discussion by mmip-labs** [akvorado/akvorado#1219] [Q&A] [answered]
Hi there!

I'm trying to receive Netflow v9 from our Cisco ASR9K, but for some reason, akvorado can't decode it. 

**My Cisco's config:**

flow exporter-map MMIP-NF-COLLECTOR
 version v9
 !
 dscp 24
 transport udp 2055
 source Loopback2
 destination 10.100.7.95 vrf MGMT

**Error in akvorado logs:**

{"level":"error","error":"NetFlowV9 NetFlowV9 [version:9 type:Decode obsDomainId:2081: templateId:260] Error template not found","exporter":"10.100.5.1","time":"2024-05-07T14:24:18Z","caller":"akvorado/inlet/flow/decoder/netflow/root.go:203","module":"akvorado/inlet/flow/decoder/netflow","message":"error while decoding NetFlow/IPFIX"}

I tried other Netflow collector (nfdump) and it could receive Netfllow successfully.

**Comment by vincentbernat** [comment]
One log message is not enough to conclude. Check the troubleshooting section of the documentation.

**Reply by mmip-labs** [reply, marked as answer]
Dear, vincentbernat
After reading https://demo.akvorado.net/docs/troubleshooting#no-packets-exported i have found my mistake ) I forgot to set default-sampling-rate in inlet.yaml Now, akvorado is working properly. Thank you for your product!

**Reply by vincentbernat** [reply]
Also see the proposed configuration for these routers (including the `sampler-table`): https://demo.akvorado.net/docs/operations#ncs-5500-and-asr-9000.

**Reply by mmip-labs** [reply]
Thank you very much!
